samples/encrypt_key.py

```diff
@@ -1,27 +0,0 @@
-import os
```

Comment:
i still think this demo has value, why don't we just refactor it to use your new Encryptor

Reply:
wrap\unwrap key is already a part of the demo.. Do you want to have standalone demo of key wrapping\unwrapping?

Reply:
Your demo is much longer and more complicated than showing a simple wrap and unwrap. I don't think it is a negative to keep it.